docs/readthedocs/models/models.md

```diff
@@ -0,0 +1,41 @@
+# Models
+
+This page gives an overview of the available models in *SIMONA*.
+
+| Model                                     | Description                    |
+|-------------------------------------------|--------------------------------|
+| [Load Model Configuration](load_model.md) | Load Model Configuration       |
+| [BM Model](bm_model)                      | Biomass plant                  |
+| [CHP Model](chp_model)                    | Combined heat and power plant  |
+| [Load Profiles](load_profiles.md)         | Load profiles                  |
+| [PV Model](pv_model.md)                   | Photovoltaic                   |
+| [Transformer Model](transformer_model.md) | Transformer Models (2W and 3W) |
+| [Thermal Storage](cts_model.md)           | Cylindrical Thermal Storage    |
+| [WEC Model](wec_model.md)                 | Wind energy converter          |
+
+```{include} load_model.md
```

Comment:
I think it's a matter of taste. What is your opinion about having each model on a separate site instead of as a waterfall thing. Maybe we can discuss in the team?